tests/integration/stop_condition_get_state_parallel/resources/scl.txt

```diff
@@ -0,0 +1,2 @@
+always get_state(actuator) = parallel;
+eventually get_state(actuator.parallel.right) = blue;
```

Comment:
Can we do an "and" to check the left state to make sure they work truly in parallel?

Reply:
Tricky. I'm not sure if this is a good idea. The parallel states operate on two independent state variables, the behaviour is checked in another tests.
The check for two paralallel states in same time is tricky and imo it does not check the implementation of stop condition.

Reply:
OK, understood. This was just a suggestion.